build_linux.sh

```diff
@@ -26,11 +26,7 @@ for d in $PLUGINS; do
 		plugin="$(basename "$d")"
 		if [ $plugin != "windows" ]; then
 			echo "  $plugin"
-                        if [ $plugin == "loopback" ]; then
-                           $GO build -tags no_openssl -o "${PWD}/bin/$plugin" "$@" "$REPO_PATH"/$d
-                        else
-                           $GO build -o "${PWD}/bin/$plugin" "$@" "$REPO_PATH"/$d
-                        fi
+			$GO build -o "${PWD}/bin/$plugin" "$@" "$REPO_PATH"/$d
```

Comment:
ummm, so we should build all with `no_openssl`, right @mrunalp ?

Reply:
FWIW, @sosiouxme had removed this in July here @ https://github.com/openshift/containernetworking-plugins/pull/12

Reply:
Yes, I think we should include that as we don't think that any of these use crypto that needs to be FIPS validated. We can maintain a list of binaries that can be excluded.